git_machete/cli.py

```diff
@@ -75,274 +77,423 @@ def usage(command: str = None) -> None:
+
+    return cli_parser
+
+
+def update_cli_opts_using_parsed_args(
+        cli_opts: git_machete.options.CommandLineOptions,
+        parsed_args: argparse.Namespace) -> None:
+    for opt, arg in vars(parsed_args).items():
+        if opt == "branch":
+            cli_opts.opt_branch = arg
+        elif opt == "checked_out_since":
+            cli_opts.opt_checked_out_since = arg
+        elif opt == "color":
+            cli_opts.opt_color = arg
+        elif opt == "down_fork_point":
+            cli_opts.opt_down_fork_point = arg
+        elif opt == "debug":
+            git_machete.options.CommandLineOptions.opt_debug = True
+        elif opt == "draft":
+            cli_opts.opt_draft = True
+        elif opt == "fetch":
+            cli_opts.opt_fetch = True
+        elif opt == "fork_point":
+            cli_opts.opt_fork_point = arg
+        elif opt == "sync_github_prs":
+            cli_opts.opt_sync_github_prs = True
+        elif opt == "inferred":
+            cli_opts.opt_inferred = True
+        elif opt == "list_commits_with_hashes":
+            cli_opts.opt_list_commits = cli_opts.opt_list_commits_with_hashes = True
+        elif opt == "list_commits":
+            cli_opts.opt_list_commits = True
+        elif opt == "merge":
+            cli_opts.opt_merge = True
+        elif opt == "n":
+            cli_opts.opt_n = True
+        elif opt == "no_detect_squash_merges":
+            cli_opts.opt_no_detect_squash_merges = True
+        elif opt == "no_edit_merge":
+            cli_opts.opt_no_edit_merge = True
+        elif opt == "no_interactive_rebase":
+            cli_opts.opt_no_interactive_rebase = True
+        elif opt == "no_push":
+            cli_opts.opt_push_tracked = False
+            cli_opts.opt_push_untracked = False
+        elif opt == "no_push_untracked":
+            cli_opts.opt_push_untracked = False
+        elif opt == "onto":
+            cli_opts.opt_onto = arg
+        elif opt == "override_to":
+            cli_opts.opt_override_to = arg
+        elif opt == "override_to_inferred":
+            cli_opts.opt_override_to_inferred = True
+        elif opt == "override_to_parent":
+            cli_opts.opt_override_to_parent = True
+        elif opt == "push":
+            cli_opts.opt_push_tracked = True
+            cli_opts.opt_push_untracked = True
+        elif opt == "push_untracked":
+            cli_opts.opt_push_untracked = True
+        elif opt == "as_root":
+            cli_opts.opt_as_root = True
+        elif opt == "roots":
+            cli_opts.opt_roots = arg.split(",")
+        elif opt == "return_to":
+            cli_opts.opt_return_to = arg
+        elif opt == "stat":
+            cli_opts.opt_stat = True
+        elif opt == "start_from":
+            cli_opts.opt_start_from = arg
+        elif opt == "unset_override":
+            cli_opts.opt_unset_override = True
+        elif opt == "verbose":
+            git_machete.options.CommandLineOptions.opt_verbose = True
```

Comment:
Hmmm does it still need to be a class attribute? given that you call `set_utils_variables` anyway, which assigns `opt_verbose`/`opt_debug` to global vars anyway